config/core/build-configs.yaml

```diff
@@ -983,6 +1019,11 @@ build_configs:
     branch: 'to-build'
     variants: *minimal_variants
 
+  bpf-next:
+    tree: bpf-next
+    branch: 'for-next'
+    variants: *bpf_variants
```

Comment:
It might be better to directly declare the variants here rather than elsewhere in the file with an anchor as it's only used with the BPF tree.